modules/ROOT/pages/servers-create.adoc

```diff
@@ -26,23 +30,25 @@ To add a Mule server to Runtime Manager:
 . From Anypoint Platform, select *Runtime Manager*.
 . Click *Servers* in the left menu.
 . Click the *Add Server* button.
+. Enter a name for your server.
++
+Server names can contain up to 60 alphanumeric characters (a-z, A-Z, 0-9), periods (.), hyphens (-), and underscores (_), but not spaces or other special characters.
+Runtime Manager supports Unicode characters in server names.
++
+The server name is unique across environments.
+If the same organization exists in different environments, the server name can be the same in those organizations.
```

Comment:
```suggestion
Although the server name must be unique across environments, it can be the same if one organization exists in different environments.
```